source/faq.html.haml.markdown

```diff
@@ -33,9 +33,9 @@ description: FAQ about the Flatpak project.
 
   ### Is Flatpak tied to Fedora?
 
-  No. The people developing Flatpak have a background in Fedora, and solving the application problem for distributions like Fedora is
+  No. In fact, Flatpak and even [Flathub](https://flathub.org) are enabled by default on Endless OS and Linux Mint.
+  The people developing Flatpak have a background in Fedora, and solving the application problem for distributions like Fedora is
```

Comment:
It might be more accurate to say that _some_ of the people developing Flatpak have a background in Fedora.  Others have a Debian background...

Reply:
"... have a long history in contributing to and building Linux distributions like Debian, Fedora and others, and solving the problems that arise when shipping applications for these platforms."?